src/platform-integration/web/wasm.md

```diff
@@ -94,52 +109,59 @@ Server started on port 8080
 
 #### Load it in a browser
 
-As of {{page.last-update}}, there are two known browser types that should run
-Flutter/Wasm content.
+As of {{page.last-update}},
+there are two known browser types that should run
```

Comment:
**issue:** This could be written with greater clarity. 


```suggestion
two browser families should be able to run
```

Reply:
Nice wording!